package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "2.0.1",
+  "version": "2.0.2",
```

Comment:
I recommend you update sbc-common-components to v42 while you're here. (The change is to retain the accountid in the URL, which was previously automatically removed by SbcHeader.)

Reply:
Good call, thanks for the heads up.

Reply:
Updated